src/test/java/org/purescript/ide/inspections/PSUnresolvedReferenceInspectionTest.kt

```diff
@@ -204,7 +204,7 @@ class PSUnresolvedReferenceInspectionTest : BasePlatformTestCase() {
             """
             module Foo where
             class Box a where
-                map :: forall a b. (a -> b) -> Box a -> Box b 
```

Comment:
I had to change this, since `Box` gave unresolved symbol errors.

Reply:
but did you not make this test useless then?

Reply:
ah ok, i first though that was the thing that got tested. but yes, that seams like a good change. Now it tests "only" what it meant to test